src/model/Simulation.js

```diff
@@ -45,6 +45,14 @@ export default class Simulation extends SimObj {
         return sim_id;
     };
 
+     async GetAllSims(user, req){
+        this.facilitator = user.id;
+        console.log(await this.toJsonObject());
+        let instances = await this.selectMany();
+        await this.select();
```

Comment:
We never use `this` after this point, so we should be able to remove this line without affecting anything